Ginger/GingerCore/Platforms/PlatformsInfo/WebPlatformInfo.cs

```diff
@@ -88,104 +83,248 @@ public override List<ActUIElement.eElementAction> GetPlatformUIElementActionsLis
+            {
+                mWebPlatformElementTypeOperations = new List<ElementTypeOperations>();
+
+                mWebPlatformElementTypeOperations.Add(new ElementTypeOperations()
+                {
+                    ElementType = eElementType.Unknown,
+                    ActionType = typeof(ActUIElement),
+                    ElementOperationsList = new List<Enum>(Enum.GetValues(typeof(ActUIElement.eElementAction)).Cast<Enum>())
+                });
+
+                mWebPlatformElementTypeOperations.Add(new ElementTypeOperations()
+                {
+                    ElementType = eElementType.Button,
+                    ActionType = typeof(ActUIElement),
+                    ElementOperationsList = new List<Enum>(){   ActUIElement.eElementAction.Click,
+                                                                ActUIElement.eElementAction.JavaScriptClick,
+                                                                ActUIElement.eElementAction.MouseClick,
+                                                                ActUIElement.eElementAction.MousePressRelease,
+                                                                ActUIElement.eElementAction.ClickAndValidate,
+                                                                ActUIElement.eElementAction.Submit,
+                                                                ActUIElement.eElementAction.GetValue,
+                                                                ActUIElement.eElementAction.IsDisabled,
+                                                                ActUIElement.eElementAction.GetFont,
+                                                                ActUIElement.eElementAction.IsVisible }
+                });
+
+                mWebPlatformElementTypeOperations.Add(new ElementTypeOperations()
+                {
+                    ElementType = eElementType.CheckBox,
+                    ActionType = typeof(ActUIElement),
+                    ElementOperationsList = new List<Enum>() {  ActUIElement.eElementAction.SetValue,
+                                                                ActUIElement.eElementAction.Click,
+                                                                ActUIElement.eElementAction.JavaScriptClick,
+                                                                ActUIElement.eElementAction.MouseClick,
+                                                                ActUIElement.eElementAction.MousePressRelease,
+                                                                ActUIElement.eElementAction.ClickAndValidate,
+                                                                ActUIElement.eElementAction.GetValue,
+                                                                ActUIElement.eElementAction.IsDisabled,
+                                                                ActUIElement.eElementAction.IsVisible }
+                });
```

Comment:
Need to add all Defualt/generic operations at the end to all ElementTypes instead again and again in each switch case, the switch case will be used for special operations types.
currently some types like "Label"/"List"/.. do not get any generic operation like "IsVisible"/...

Reply:
above still was not done